packages/web/src/javascripts/Components/Tags/TagsList.tsx

```diff
@@ -6,16 +6,26 @@ import { TagListSectionType } from './TagListSection'
 import { TagsListItem } from './TagsListItem'
 import { useApplication } from '../ApplicationProvider'
 import { useListKeyboardNavigation } from '@/Hooks/useListKeyboardNavigation'
+import { NavigationController } from '@/Controllers/Navigation/NavigationController'
 
 type Props = {
   type: TagListSectionType
 }
 
+function getAllTagsForType(controller: NavigationController, type: TagListSectionType) {
```

Comment:
I'd suggest either just passing `isSearching` here as a parameter instead of the entire controller object, or just have this function read from `application`, that way it'd be clearer to see what input the function needs